common/templates/form-wizard.njk

```diff
@@ -32,7 +32,7 @@
   }) }}
 
   <header class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
```

Comment:
Changed this so the heading "Search for a person to move" doesn't wrap - again to match the wireframe.